reports/report.md

```diff
@@ -2608,6 +2659,18 @@ Consider using a specific version of Solidity in your contracts instead of a wid
 	pragma solidity ^0.4.0;
 	```
 
+- Found in src/StateVariableCouldBeDeclaredConstant.sol [Line: 2](../tests/contract-playground/src/StateVariableCouldBeDeclaredConstant.sol#L2)
```

Comment:
We should follow the advice that aderyn gives. We should avoid writing new contracts with known issues that are captured by other detectors, unless it's unavoidable.

* For example, giving the contract a specific pragma can avoid this open pragma issue.
* An example of an unavoidable one is "Unused State Variables".